shell/platform/darwin/ios/framework/Source/TextInputSemanticsObject.mm

```diff
@@ -462,4 +462,34 @@ - (BOOL)hasText {
   return [[self textInputSurrogate] hasText];
 }
 
+#pragma mark - UIResponder overrides
+
+- (void)cut:(id)sender {
```

Comment:
I'm not sure all the gotchas, but can this class use `forwardingTargetForSelector` instead of directly calling every method on `textInputSurrogate`? It's inevitable we will miss some as they are added in future SDKs.  I'm not sure it's the right behavior for this class, though, and I'm not sure if the super class `UIAccessibilityElement` already implements, in which case it won't work...

Reply:
It turns out that iOS determines whether to show these options in accessibility rotor by checking the presence of these functions, So we can't use `forwardingTargetForSelector`.